chrome/content/zotero/xpcom/utilities.js

```diff
@@ -311,7 +311,7 @@ Zotero.Utilities = {
 			throw "cleanDOI: argument must be a string";
 		}
 
-		var doi = x.match(/10\.[0-9]{4,}\/[^\s]*[^\s\.,]/);
+		var doi = x.match(/(?:10\.[0-9]{4,}\/[^\s]*[^\s\.,])|(?:10\/[^\s]*[^\s\.,])/);
```

Comment:
Seems like we could shorten this to `(?:10(?:\.[0-9]{4,})?\/[^\s]*[^\s\.,])`? As in, just make the `.<number>` between the `10` and the `/` optional?

Reply:
Good point. Made that change.